airflow-core/src/airflow/models/taskmap.py

```diff
@@ -131,14 +131,14 @@ def expand_mapped_task(cls, task, run_id: str, *, session: Session) -> tuple[Seq
             order by map index, and the maximum map index value.
         """
         from airflow.models.expandinput import NotFullyPopulated
-        from airflow.models.mappedoperator import get_mapped_ti_count
+        from airflow.models.mappedoperator import MappedOperator as DbMappedOperator, get_mapped_ti_count
```

Comment:
Should we unify the way we name the ones from `sdk.definitions`, `sdk.types`, or `models` and document it? Probably don't need to be done in this PR, but it could be a bit confusing when reading through different modules.